cli/tsc/mod.rs

```diff
@@ -165,32 +133,128 @@ pub static LAZILY_LOADED_STATIC_ASSETS: Lazy<
   HashMap<&'static str, &'static str>,
 > = Lazy::new(|| {
   ([
-    (
-      "lib.dom.asynciterable.d.ts",
-      inc!("lib.dom.asynciterable.d.ts"),
+    ("lib.deno.webgpu.d.ts", inc!("lib.deno_webgpu.d.ts")),
+    ext_lib!("lib.deno.console.d.ts", "console/lib.deno_console.d.ts"),
```

Comment:
can't use `deno_console::get_declaration()` because we need a string literal for `include_str!` to work